vhdbuilder/packer/trivy-scan.sh

```diff
@@ -3,7 +3,7 @@ set -euxo pipefail
 
 TRIVY_REPORT_JSON_PATH=/opt/azure/containers/trivy-report.json
 TRIVY_REPORT_TABLE_PATH=/opt/azure/containers/trivy-images-table.txt
-TRIVY_VERSION="0.40.0"
+TRIVY_VERSION="0.53.0"
```

Comment:
why the bump here?

Reply:
Just looking to keep current, I can bring it back down if necessary. I originally bumped in order to make use of parallel scanning but that was a dead end.

I reached out to Jason because I know he is working with the trivy scanning a bit and he said he didn't have a preference.

Reply:
as long as things still work I'm fine with it, just wanted to check